Src/Base/AMReX_FabArrayBase.cpp

```diff
@@ -718,7 +733,9 @@ FabArrayBase::FB::define_fb(const FabArrayBase& fa)
     {
 	const int   krcv = imap[i];
 	const Box& vbx   = ba[krcv];
+	const Box& vbx_ng  = amrex::grow(ba[krcv],1);
 	const Box& bxrcv = amrex::grow(vbx, ng);
+	const Box& bxrcv_ng = amrex::grow(vbx_ng, ng_ng);
```

Comment:
Looks to me `bxrcv_ng == bxrcv`.  So we can remove this.

Reply:
Done